src/lwc/geFormRenderer/geFormRenderer.js

```diff
@@ -1997,15 +1996,20 @@ export default class GeFormRenderer extends LightningElement{
     hasChargeableTransactionStatus = () => {
         const paymentStatus = this.getFieldValueFromFormState(PAYMENT_STATUS);
         switch (paymentStatus) {
```

Comment:
I'd like to simplify this switch statement a bit. We're returning either True or False here, we can test for the `True` criteria and return false otherwise. The only reason I can see that we may not want to do this is if we want to explicitly call out the other statuses that we're returning false for. 

I think `hasChargeableTransactionStatus` could also be pared down to two lines, by moving the switch statement to a new function called `isChargeableStatus(paymentStatus)`.

```
switch (paymentStatus) {
  case '':
  case undefined:
  case null:
  case this.PAYMENT_TRANSACTION_STATUS_ENUM.PENDING:
  case this.PAYMENT_TRANSACTION_STATUS_ENUM.DECLINED:
  case this.PAYMENT_TRANSACTION_STATUS_ENUM.RETRYABLEERROR:
    return true;

  default:
    return false;
}
```

Reply:
@bdvorachek updated!